ufersa/plataforma-sabia
# Create ideas bank page

## Summary


Resolves #537 

## Relevant technical choices


- Environment variables were updated
- Algolia indexes for specific environments now end with said environment name (e.g. `searchable__technology__production`)

## QA Steps


1. Update your `.env` (see `.env.example`)
2. At home page, click `recursos > banco de ideias`
3. The page is responsive
4. It's possible to add a new idea
5. All ideas are listed and have instant search by Algolia

## Checklist
